containers/ruby/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine:3.6
+
+RUN apk add --no-cache ruby bash
```

Comment:
should install a particular version of ruby. otherwise the image won't stay consistent over time